syntax = "proto3";

import public "image.proto";
import public "alerts.proto";

package visualization;

service VisualizationService {
  rpc HandleImageNotif(img_common.CameraFrame) returns (common.Result) {}
  rpc HandleAlertNotif(alerts.Alert) returns (common.Result) {}
}
